LICENSE.MIT

```diff
@@ -0,0 +1,30 @@
+------------------------------------------------------------------------------
+This license applies to the following third-party libraries included
+in this repository:
+
+  - backbone.paginator
+  - Backbone.js
+  - Underscore.js
```

Comment:
@bradenmacdonald It's not necessary to include Backbone and Underscore when running problem-builder inside LMS/Studio, I just checked. However, tests fail horribly without them. Is there a way to conditionally include them, i.e., load them iff problem-builder is running as a standalone XBlock?